stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Any.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.chat.android.ui.utils.extensions
+
+internal inline fun <reified AnyT> Any.cast() = this as AnyT
```

Comment:
Maybe just `T` instead of `AnyT` parameter?

Reply:
I generally prefer naming generic parameters with some sort of contextual name. We don't use single letter variable names, so why do it with generics? Makes the code easier to read and understand, in my opinion. Anyway, I can change it if we feel strongly about it.